db/bootstrap.py

```diff
@@ -0,0 +1,176 @@
+"""
+Serves as the entrypoint of the database container, running migrations as
+necessary and automatically running new migrations as they appear.
+"""
+
+__author__ = "Architus"
+__version__ = "0.1.0"
+
+import os
+import sys
+import shlex
+import argparse
+import subprocess
+
+POSTGRES_USER = "postgres"
+BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+MIGRATIONS_DIR = os.path.join(BASE_DIR, "./migrations")
+CURRENT_MIGRATIONS_DIR = os.path.join(BASE_DIR, "./current_migration")
+CURRENT_MIGRATION_FILE = "current.out"
+ORIGINAL_ENTRYPOINT = "/usr/local/bin/docker-entrypoint.sh"
+INJECTED_ENTRYPOINT = os.path.join(BASE_DIR, "./entrypoint.sh")
+
+
+def main():
+    print("Running db migration manager")
+    current_migration = get_current_migration()
+    to_run = get_new_migrations(current_migration)
+    if to_run:
+        print(f"Running migrations: [{', '.join(to_run)}]")
```

Comment:
isn't this the exact same behavior as `f"Running migrations: {to_run}"`

Reply:
probably